xbmc/messaging/ThreadMessage.h

```diff
@@ -59,10 +59,9 @@ class ThreadMessage
     , param1{ p1 }
     , param2{ p2 }
     , lpVoid{ payload }
-
+    , strParam{ param }
+    , params{ vecParams }
```

Comment:
This might break, stupid me not commenting but I think they're in the body because vs2013 had some trouble with having them in the init list.